docs/cookbook/loading_molecules.md

```diff
@@ -0,0 +1,93 @@
+# Loading your data
+
+One of the first tasks you'll likely want to do is loading your various input files.
+In ``openfe`` the entire contents of a simulation volume, for example the ligand, protein and water is referred to
+as the ``ChemicalSystem``.
+A free energy difference is defined as being between two such ``ChemicalSystem`` objects.
+To make expressing free energy calculations easier,
+this ``ChemicalSystem`` is broken down into various ``Component`` objects.
+It is these ``Component`` objects that are then transformed, added or removed when performing a free energy calculation.
+
+
+:::{note}
+Once chemical models are loaded into Components they are **read only** and cannot be modified.
+This means that any modification/tweaking of the inputs must be done **before** the Component objects are created. 
+This is done so that any data cannot be accidentally modified, ruining the provenance chain.
+:::
+
+As these all behave slightly differently to accomodate their contents,
+there are specialised versions of Component to handle the different items in your system.
+We will walk through how different items can be loaded,
+and then how these are assembled to form ``ChemicalSystem`` objects.
+
+
+## Small molecules
+
+Small molecules, such as ligands, are handled using the ``openfe.SmallMoleculeComponent`` class.
+These are lightweight wrappers around RDKit Molecules and can be created directly
+from an rdkit molecule:
+
+```python
+from rdkit import Chem
+import openfe
+
+m = Chem.MolFromMol2File('myfile.mol2', removeHs=False)
+
+smc = openfe.SmallMoleculeComponent(m, name='')
+```
+
+:::{caution}
+Remember to include the `removeHs=False` keyword argument so that RDKit does not strip them!
```

Comment:
```suggestion
Remember to include the `removeHs=False` keyword argument so that RDKit does not strip your hydrogens!
```
(might be obvious by its name, but in practice folks might just not know rdkit that well)